app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -126,6 +146,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         });
       }
 
+      function preventLinkFollowingForCurrentTab(event) {
```

Comment:
It doesn't act only as a toggle. If you open the link in a new tab, it will scroll you to the correct expanded accordion section (same as if you bookmark the section, or share it)